zebra/zserv.h

```diff
@@ -90,6 +90,9 @@ struct zserv {
 	/* Redistribute default route flag. */
 	vrf_bitmap_t redist_default[AFI_MAX];
 
+	/* Interface information. */
```

Comment:
This really is vrf's we are interested in.  Let's name it appropriately

Reply:
I will change this to vrf_reg_info.